From a6c5ce674dc285b8c95d4aed6543e81df7193db9 Mon Sep 17 00:00:00 2001
From: "Yong, Jonathan" <jonathan.yong@intel.com>
Date: Tue, 26 Feb 2019 02:53:41 +0000
Subject: [PATCH] pam-ck-connector: fix srcdir != builddir build

The original Makefile assumes in-tree build, correct the
variable used.

Signed-off-by: Yong, Jonathan <jonathan.yong@intel.com>
---
 pam-ck-connector/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pam-ck-connector/Makefile.am b/pam-ck-connector/Makefile.am
index 7778d16..9b960a7 100644
--- a/pam-ck-connector/Makefile.am
+++ b/pam-ck-connector/Makefile.am
@@ -4,7 +4,7 @@ if ENABLE_PAM_MODULE
 
 AM_CFLAGS = 					\
 	$(LIBDBUS_CFLAGS)			\
-	-I$(top_builddir)/libck-connector	\
+	-I$(top_srcdir)/libck-connector	\
 	$(NULL)
 
 pamlibdir = $(PAM_MODULE_DIR)
-- 
2.17.1

